blocks/embedded-form/embedded-form.js

```diff
@@ -1,7 +1,6 @@
 import { div } from '../../scripts/scripts.js';
```

Comment:
Part of the refactor was moving the common form related functions (validate, submit, etc) over to a single forms.js file and exporting those methods)